tools/radar_sync.py

```diff
@@ -0,0 +1,404 @@
+                        help=('prints debug messages'))
+    parser.add_argument('--hdfs', action='store', type=str, required=True,
+                        dest='hdfs_url',
+                        help=(
+                            'hdfs server and path of the form: '
+                            'hdfs://<NAME_NODE>:<PORT>/PATH'))
+    parser.add_argument('--ssh', action='store', type=str, required=True,
+                        dest='ssh_url',
+                        help=(
+                            'ssh server and path of the form: '
+                            '<USER>@<NAME_NODE>:<PORT>/PATH'))
+    parser.add_argument('--key', action='store', type=str, required=True,
+                        dest='ssh_key',
+                        help=('key for ssh server authentication'))
+
+    args = parser.parse_args()
+
+    # If the debug flag is set, print all messages
+    if args.debug:
+        logging.basicConfig(
+            level=logging.DEBUG,
+            format='[%(levelname)s] %(message)s')
+    else:
+        logging.basicConfig(
+            level=logging.INFO,
+            format='[%(levelname)s] %(message)s')
+
+    logging.getLogger("paramiko").setLevel(logging.WARNING)
+
+    _hdfs_url = check_hdfs_url(args.hdfs_url)
+    if _hdfs_url is None:
+        logging.error(
+            'Wrong, incomplete or absent HDFS path: \'%s\'', args.hdfs_url)
+        sys.exit(1)
+
+    (_ssh_username, _ssh_hostname, _ssh_port,
+     _ssh_root) = check_ssh_url(args.ssh_url)
+    if _ssh_hostname is None:
+        logging.error(
+            'Wrong, incomplete or absent SSH path: \'%s\'', args.ssh_url)
+        sys.exit(1)
+
+    _hdfs_remote_files = list_hdfs_files(_hdfs_url)
+
+    ssh_client = SSHClient(
+        username=_ssh_username,
+        hostname=_ssh_hostname,
+        port=_ssh_port,
+        key_file=args.ssh_key,
+        root_dir=_ssh_root
+    )
+
+    _ssh_days_to_retrieve = ssh_hdfs_folder_diff(
+        ssh_client,
+        _hdfs_remote_files)
+
+    logging.info('There are %d folders to copy from SFTP to HDFS',
+                 len(_ssh_days_to_retrieve))
+
+    for _ssh_day_to_retrieve in _ssh_days_to_retrieve:
+        _ssh_files = ssh_client.list_folder(_ssh_day_to_retrieve)
+        _local_path = os.path.basename(_ssh_day_to_retrieve).lstrip('/')
+        _hdfs_path = f'{_hdfs_url}/{_ssh_day_to_retrieve}'
+
+        logging.info(
+            "Retrieving SFTP folder '%s', %d files, %dM to local folder '%s'",
+            _ssh_day_to_retrieve, len(_ssh_files),
+            ssh_client.get_folder_size(_ssh_day_to_retrieve),
+            _local_path)
+
+        if not args.dryrun:
+            ssh_client.remote_to_local_copy(_ssh_day_to_retrieve)
+
+        logging.info(
+            "Copying local folder '%s' to HDFS folder '%s'",
+            _local_path, _hdfs_path)
+        if not args.dryrun:
+            copy_to_hdfs(_local_path, _hdfs_path)
+
+        delete_local_folder(_local_path, args.dryrun)
```

Comment:
Looks like we can just use `shutil.rmtree` here, and get rid of `delete_local_folder`